stable/redis/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+Redis can be accessed via port 3306 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To get your password run:
+
+    REDIS_PASSWORD=$(printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.redis-password[*]}"`))
```

Comment:
I commented on this in another PR but we should be leveraging the features that are intended to solve the use case for passwords, namely secrets. The authentication and encryption issues will eventually solved in the Secrets resource but are much less likely to be fixed in ConfigMaps.